sdk/test-utils/recorder/README.md

```diff
@@ -45,74 +45,97 @@ tests of an sdk.
 
-We're about to go through how to set up your project to use the test-utils-recorder.
-We'll be using git and [rush](https://rushjs.io). We understand that both tools
-can be challenging to use, but we expect you to know how to use them by the
-time you decide to use this package. If you encounter any problem with these
-tools, please make sure to check at their documentation first, but we're also
-happy to answer any question! We appreciate that you're reading this 💙
+We're about to go through how to set up your project to use the
+`@azure/test-utils-recorder` package.
+
+This document assumes familiarity with [git](https://git-scm.com) and [rush](https://rushjs.io).
 
-Keep in mind that test-utils-recorder is not a published package. It is only intended to be used
-by our libraries (at least for now).
+Keep in mind that `@azure/test-utils-recorder` is not a published package. It
+is only intended to be used by the libraries in the azure-sdk-for-js repository
+(at least for now).
 
 ### Installing the package
 
-To install the test-utils-recorder package, you'll need to start by cloning our
-azure-sdk-for-js repository. One way of doing this is by using the git command
-line interface, as follows:
+To install the `@azure/test-utils-recorder` package, you'll need to start by
+cloning our azure-sdk-for-js repository. One way of doing this is by using the
+git command line interface, as follows:
 
-    cd /path/to/my/github/repositories
-    git clone https://github.com/Azure/azure-sdk-for-js/
+```bash
+cd /path/to/my/github/repositories
+git clone https://github.com/Azure/azure-sdk-for-js/
+```
 
 Having cloned this repository, let's set it up by running the following rush commands:
 
-    cd azure-sdk-for-js
-    rush update
-    rush install
-    rush build
+```bash
+cd azure-sdk-for-js
+rush update
+rush install
+rush build
+```
 
 This will optimistically assume you're in a fresh clone.
 
 From this point forward, we'll assume that you're developing (perhaps
-contributing!) to one of our libraries. So, your next step is to change
-directory to the path relevant to your project. Let's say you want to add
-test-utils-recorder to the package `@azure/keyvault-keys` (it already uses test-utils-recorder,
-but bear with us), you'll be doing the following:
-
-    cd sdk/keyvault/keyvault-keys
+contributing!) to one of the azure-sdk-for-js's libraries. So, your next step
+is to change directory to the path relevant to your project. Let's say you want
+to add the `@azure/test-utils-recorder` package to `@azure/keyvault-keys` (it
+already uses test-utils-recorder, but bear with us), you'll be doing the
+following:
+
+```bash
+cd sdk/keyvault/keyvault-keys
+```
 
-One there, you can add the test-utils-recorder package by running the following rush command:
+Once there, you can add the test-utils-recorder package by running the following rush command:
 
-    rush add -p @azure/test-utils-recorder
+```bash
+rush add -p @azure/test-utils-recorder
```

Comment:
I'm not sure if `rush add` actually works for this package.
Can you please double-check?